cristin-import/src/test/java/cucumber/EventFeatures.java

```diff
@@ -104,6 +105,15 @@ public void theNvaEventHasAPeriodWithFromDate(String value) {
         assertThat(((Period) event.getTime()).getFrom().toString(), is(equalTo(value)));
     }
 
+    @Then("the Event has no a time Period with toDate")
```

Comment:
Suggestion: Rename this to "the Event toDate is null" or "the Event end date is missing" perhaps? Equivalent for the method name could be "theNvaEventPeriodToDateEquals".